dev-docs/bidders/acuityads.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: bidder
+title: AcuityAds
+description: Prebid AcuityAds Bidder Adaptor
+biddercode: acuityads
+gdpr_supported: true
+tcf2_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+userId: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+media_types: banner, video, native
+safeframes_ok: true
+bidder_supports_deals: true
+pbjs: true
```

Comment:
I don't see a PR for a Prebid.js bid adapter. Please either link in the PR or set pbjs: false. Thanks.

Reply:
@bretg Thank you! My bad. I have fixed it.